### PR TITLE
Fix extra SQL support in Doctrine migrations

### PR DESCRIPTION
**Pull request by fabpot**
| Q             | A
| ------------- | ---
| Branch?       | 5.1
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | n/a
| License       | MIT
| Doc PR        | n/a

/cc @weaverryan

### CODE REVIEW

File: src/Symfony/Bridge/Doctrine/SchemaListener/MessengerTransportDoctrineSchemaSubscriber.php [hunk 1]
```diff
@@ -79,7 +78,9 @@ public function onSchemaCreateTable(SchemaCreateTableEventArgs $event): void
              * the only way to inject some extra SQL.
              */
             $event->addSql($createTableSql);
-            $event->addSql($extraSql);
+            foreach ($extraSql as $sql) {
+                $event->addSql($sql);
```
Comment (fabpot) [inline src/Symfony/Bridge/Doctrine/SchemaListener/MessengerTransportDoctrineSchemaSubscriber.php:82]: This is the fix: `addSql()` onlt works if it is given a single SQL statement.

File: src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/DoctrineTransport.php [hunk 2]
```diff
@@ -111,8 +111,10 @@ public function configureSchema(Schema $schema, DbalConnection $forConnection):
 
     /**
      * Adds extra SQL if the given table was created by the Connection.
+     *
+     * @return []string
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/DoctrineTransport.php:115]: `@return string[]` (same below)

Comment (fabpot) [reply src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/DoctrineTransport.php:115]: Too much Go these days :)

**Comment by fabpot** [comment]
Tests fail are expected.

**Review by nicolas-grekas — APPROVED** [review]

**Pull request merged**